### PR TITLE
initial set up for UI

### PR DESCRIPTION
**Pull request by olivena**
There is bunch of stuff that does not belong there and I have to edit it still. Just wanted to push it to git for our weekly meeting.

### CODE REVIEW

**Comment by rvsape** [comment]
Alright. But you can push into git without making a PR too :)

**Comment by OpaJaa** [comment]
You could pull main branch to this and add a readme to /cc-ui/ with usage directions.

**Comment by bunjovis** [comment]
@olivena are you still going to add code to this pull request before we should approve (based on your comment)? let us know when its ready

**Comment by olivena** [comment]
I cleaned the code a bit, if its okay that i add the readme after merging, it is okay to merge

**Comment by bunjovis** [comment]
No, add the readme before.

**Comment by OpaJaa** [comment]
> I cleaned the code a bit, if its okay that i add the readme after merging, it is okay to merge

You would need to do a new pull request for the readme or add it in a later one as you cannot push changes straight to main. So a readme with the info needed at this time should be added. Like how to run the service etc.

File: cc-ui/src/App.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.logo {
```
Comment (hannakortetjarvi) [inline cc-ui/src/App.css]: You can make this App.css file look cleaner by removing the parts that aren't used in the program. For example, I don't think logo was mentioned in the code anywhere

**Review by hannakortetjarvi — COMMENTED** [review]
I think you can also remove the react.svg -file, since you are using vite.svg as a favicon

**Comment by olivena** [comment]
I think the initial set up for UI is finally ready, I added readme and did more cleaning up, im sorry I didn't have time last week to do these final touches.

File: cc-ui/src/main.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import App from './App'
+import * as ReactDOM from 'react-dom'
+import { ChakraProvider } from '@chakra-ui/react'
+import './index.css'
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <React.StrictMode>
```
Comment (rvsape) [inline cc-ui/src/main.tsx:8]: Should the App-component be wrapped with ChakraProvider?

File: cc-ui/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './App.css'
+import { ChakraProvider } from '@chakra-ui/react'
```
Comment (rvsape) [inline cc-ui/src/App.tsx]: unused import

File: cc-ui/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "cc-ui",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
```
Comment (rvsape) [inline cc-ui/package.json:11]: chakra-ui is missing

Comment (rvsape) [reply cc-ui/package.json:11]: npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion

**Review by rvsape — CHANGES_REQUESTED** [review]
Chakra-UI is missing from packages, and is not used correctly

File: cc-ui/src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+function App() {
+  return (
+    <ChakraProvider>
```
Comment (rvsape) [inline cc-ui/src/App.tsx]: It is enough to use the ChakraProvider wrapper in the main.tsx, so it can be removed here

**Review by rvsape — CHANGES_REQUESTED** [review]
Duplicate use of ChakraProvider. It is enough to use it only in the main.tsx file

**Review by rvsape — APPROVED** [review]

**Review by hannakortetjarvi — APPROVED** [review]

**Pull request merged**